### ISSUE TITLE
Unusual structure nsubj->obl->root

### ISSUE THREAD

**Issue by ftyers** [UniversalDependencies/UD_Russian-SynTagRus#8]
Does this structure look right ? Is it possible to have a subject of something that isn't a clause ?

```
# source = 2006Vtoraya_popytka.xml 157
# text = А что до источников роста, то в Грузии рентного роста не будет, слава богу.
# sent_id = 12696
1       А       а       CCONJ   _       _       9       cc      9:cc    _
2       что     что     PRON    _       Case=Nom        4       nsubj   4:nsubj _
3       до      до      ADP     _       _       4       case    4:case  _
4       источников      источник        NOUN    _       Animacy=Inan|Case=Gen|Gender=Masc|Number=Plur   9       obl     9:obl   _
5       роста   рост    NOUN    _       Animacy=Inan|Case=Gen|Gender=Masc|Number=Sing   4       nmod    4:nmod  SpaceAfter=No
6       ,       ,       PUNCT   _       _       5       punct   5:punct _
7       то      то      SCONJ   _       _       2       cc      2:cc    _
8       в       в       ADP     _       _       9       case    9:case  _
9       Грузии  грузия  PROPN   _       Animacy=Inan|Case=Loc|Gender=Fem|Number=Sing    0       root    0:root  _
10      рентного        рентный ADJ     _       Case=Gen|Degree=Pos|Gender=Masc|Number=Sing     11      amod    11:amod _
11      роста   рост    NOUN    _       Animacy=Inan|Case=Gen|Gender=Masc|Number=Sing   9       nsubj   9:nsubj _
12      не      не      PART    _       _       9       advmod  9:advmod        _
13      будет   быть    VERB    _       Aspect=Imp|Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin|Voice=Act      9       cop     9:cop   SpaceAfter=No
14      ,       ,       PUNCT   _       _       13      punct   13:punct        _
15      слава   слава   NOUN    _       Animacy=Inan|Case=Nom|Gender=Fem|Number=Sing    9       parataxis       9:parataxis     _
16      богу    бог     NOUN    _       Animacy=Anim|Case=Dat|Gender=Masc|Number=Sing   15      fixed   15:fixed        SpaceAfter=No
17      .       .       PUNCT   _       _       16      punct   16:punct        _

```

**Comment by dan-zeman** [comment]
It does not look right. Using `nsubj` to attach _что_ implies that _что до источников роста_ is a clause (with a non-verbal predicate). That is, the relation between _Грузии_ and _источников_ must be `advcl` instead of `obl`.

**Comment by ftyers** [comment]
That's my vote too. Also, should то here be attached to 'что' ? and not to 'источников' or to 'Грузии'?

**Comment by dan-zeman** [comment]
I think it should be attached to _Грузии_.

**Comment by ftyers** [comment]
The question is if this can be done given the existing annotation, @Kira-D what does the SynTagRus tree look like here ?

**Comment by Kira-D** [comment]
Forgot to apply the fix to 2.1, but it's in dev brunch now

**Comment by olesar** [comment]
The construction ЧТО ДО X.Gen is a case of ellipsis. There are two possible alternatives "Что касается источников роста" and more old-fashioned "Что касается до источников роста" (cf. RNC examples: http://search1.ruscorpora.ru/search.xml?env=alpha&mycorp=&mysent=&mysize=&mysentsize=&mydocsize=&spd=&text=lexgramm&mode=main&sort=gr_tagging&lang=ru&nodia=1&parent1=0&level1=0&lex1=%F7%F2%EE&gramm1=&sem1=&sem-mod1=sem&sem-mod1=sem2&flags1=&m1=&parent2=0&level2=0&min2=1&max2=1&lex2=%EA%E0%F1%E0%F2%FC%F1%FF&gramm2=&sem2=&sem-mod2=sem&sem-mod2=sem2&flags2=&m2=&parent3=0&level3=0&min3=1&max3=1&lex3=%E4%EE&gramm3=&sem3=&sem-mod3=sem&sem-mod3=sem2&flags3=&m3=)
So, this is definitely a clause, which is to be analysed according the rules for ellipsis.
